### PR TITLE
Calendar header

### PR DESCRIPTION
**Pull request by eddielo91**

### CODE REVIEW

**Comment by Jaguirre123** [comment]
let me know when its ready for review

**Comment by eddielo91** [comment]
It's ready was just fixing some more stuff while its here haha

File: __test__/comnponent.test.js [hunk 1]
```diff
@@ -60,10 +63,62 @@ describe('React Component Testing', () => {
...
+        const mockClickHandler = jest.fn();
+        const wrapper = shallow(
+          <CalendarHeader 
+            calendarDate={new Date('Wednesday, September 19, 2018')}
+            handleIncreaseMonthClick={mockClickHandler}
+            handleDecreaseMonthClick={mockClickHandler}
+          />);
+        wrapper.find('#forward').simulate('click');
+        wrapper.find('#back').simulate('click');
+        expect(mockClickHandler.mock.calls.length).toBe(2);
```
Comment (Windsvept) [inline __test__/comnponent.test.js:106]: Nice tests!

File: client/sidebar.css [hunk 2]
```diff
@@ -163,10 +175,83 @@ select {
...
+.forward:focus, .back:focus {
+  outline: none;
+}
+
+.days-of-week {
+  font-family: Helvetica;
+  font-size: 12px;
+  color: #d32300;
+  padding-top: 7.5px;
+  cursor: default;
```
Comment (Windsvept) [inline client/sidebar.css:256]: O wow are you done with calendar?

**Review by Windsvept — APPROVED** [review]

File: client/sidebar.css (hunk 2 above)
Comment (eddielo91) [reply client/sidebar.css:256]: Just the calendar header haha

**Pull request merged**